src/claim.rs

```diff
@@ -0,0 +1,83 @@
+use chrono::prelude::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+/// Implementation of CNAB Claims 1.0
+///
+/// This provides a struct that matches the CNAB Claims 1.0 specification at the
+/// time when the CNAB Core 1.0 specification was finalized.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Claim {
+    /// The bundle descriptor
+    pub bundle: crate::cnab::Bundle,
+    /// Creation date
+    pub created: DateTime<Utc>,
+    /// Extension space
+    pub custom: Option<serde_json::Value>,
+    /// Modification date
+    pub modified: DateTime<Utc>,
+    /// The name of the claim (e.g. the release name)
+    pub name: String,
+    /// Name/value pairs representing the outputs from the runtime
+    pub outputs: Option<BTreeMap<String, String>>,
+    /// Name/value paris that represent the parameter values
+    pub parameters: Option<BTreeMap<String, String>>,
```

Comment:
When constructing a claim, are users expected to serialize the parameters before-hand?
Should this be a `Option<BTreeMap<String, Parameter>>`?